examples/zkapps/04-zkapp-browser-ui/contracts/src/Add.test.ts

```diff
@@ -8,6 +8,7 @@ import {
   PublicKey,
   AccountUpdate,
 } from 'snarkyjs';
+import { LocalBlockchain } from 'snarkyjs/dist/node/lib/mina';
```

Comment:
this should be imported from `'snarkyjs'`

Reply:
Ah, thanks. Looks like it can be removed completely!